lib/features/popup-menu/PopupMenuComponent.js

```diff
@@ -88,8 +88,9 @@ export default function PopupMenuComponent(props) {
     return searchFn(searchableEntries, value, {
       keys: [
         'label',
-        'description',
-        'search'
+        'search',
+        'keywords',
```

Comment:
If we start so source `keywords`, then we need test cases for it. However I don't see why we'd need this (and instead rely on `search=string[]` instead.

https://github.com/bpmn-io/bpmn-js-create-append-anything/pull/50#pullrequestreview-2729084390

Reply:
What we want to do is to extend the popup menu entry types, so they include `search=string|string[]`.